recipes/sass_setup.rb

```diff
@@ -0,0 +1,9 @@
+remove_file 'app/assets/stylesheets/application.css'
+create_file 'app/assets/stylesheets/application.scss'
```

Comment:
The default is to `require_tree .`, but it looks like we are favoring being explicit with our imports in our [raygun template](https://github.com/spartansystems/raygun-rails-template/blob/master/app/assets/stylesheets/application.scss).